NuGet.config

```diff
@@ -1,8 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="Aspnet Release" value="https://myget.org/f/aspnetrelease" />
+    <clear />
+    <add key="Aspnet Release" value="https://www.myget.org/F/aspnetrc2/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="msextensions" value="\\aspnetci\drops\Coherence-Signed\release\16795\artifacts\Signed\Packages-NoTimeStamp" />
+    <add key="Aspnet Release Old" value="https://myget.org/f/aspnetrelease" />
```

Comment:
I had some issues with respect to a package, and this resolved the issue. We can still have it, as the final bits are not ready yet from their side. Will clean this up as soon as we are at the end.